File: easy/4_add_numbers/inverted_star.js
function invertedStar(n) {
	for (let i = 0; i <= n; i++) {
		for (let j = 0; j < n-i; j++) {
			process.stdout.write(" ")
		}
		for (let k = 0; k < i; k++) {
			process.stdout.write("*")
		}
		process.stdout.write("\n")
	}
}

invertedStar(5)

module.exports = invertedStar;

File: easy/3_inverted_star/inverted_star.go
package main

import "fmt"

func invertedStar(n int) {
	for i := 0; i <= n; i++ {
		for j := 0; j < n-i; j++ {
			fmt.Printf(" ")
		}
		for k := 0; k < i; k++ {
			fmt.Printf("*")
		}
		fmt.Println()
	}
}

func main() {
	invertedStar(5)
}
